index.html

```diff
@@ -4,6 +4,7 @@
     <title>מפתח התקציב - חיפוש</title>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <base href="/">
```

Comment:
I think not. I removed the <base href> tag in a new PR as we don't need it for now. The router is configured to used the old strategy of hash location strategy that does not need it. 

https://github.com/OpenBudget/budgetkey-app-search/pull/57

If we do need <base> when router strategy will change to html5 path location strategy, to support dev and prod environment build we could use 

    <script>document.write('<base href="' + document.location + '" />');</script>

Or doing something in the build phase for production that build and specify specifically with the base href as param like angular cli does:
https://github.com/angular/angular-cli/wiki/build#base-tag-handling-in-indexhtml

> ### Sets base tag href to /myUrl/ in your index.html
> ng build --base-href /myUrl/
> ng build --bh /myUrl/